src/Protocol/Http/Authenticator/JwtAuthenticator.php

```diff
@@ -40,14 +55,37 @@ public function __construct(JwtInterface $jwt)
         $this->jwt = $jwt;
     }
 
+    /**
+     * Get jws lifetime
+     *
+     * @return integer
+    */
+    public function getJwsLifetime() : int
+    {
+        return $this->jwsLifetime;
+    }
+
+    /**
+     * Set jws lifetime
+     *
+     * @param integer $jwsLifetime
+     */
+    public function setJwsLifetime(int  $jwsLifetime)
```

Comment:
Maybe better use the constructor for injecting the lifetime of token?

We have the solution where we should set the new value of expiring in runtime process? Yes - all ok, No - pass from the constructor.